tests/cypress/integration/actions_tasks_objects/case_20_objects_ordering_feature.js

```diff
@@ -0,0 +1,83 @@
+// Copyright (C) 2020 Intel Corporation
+//
+// SPDX-License-Identifier: MIT
+
+/// <reference types="cypress" />
+
+import { taskName, labelName, attrName } from '../../support/const';
+
+context('Objects ordering feature', () => {
+    const caseId = '20';
+
+    const createRectangleShape2Points = {
+        points: 'By 2 Points',
+        type: 'Shape',
+        labelName: labelName,
+        firstX: 250,
+        firstY: 350,
+        secondX: 350,
+        secondY: 450,
+    };
+
+    const createRectangleShape2PointsSecond = {
+        points: 'By 2 Points',
+        type: 'Shape',
+        labelName: labelName,
+        firstX: createRectangleShape2Points.firstX + 300,
+        firstY: createRectangleShape2Points.firstY,
+        secondX: createRectangleShape2Points.secondX + 300,
+        secondY: createRectangleShape2Points.secondY,
+    };
+
+    function idAscent(arr) {
+        for (let i = 0; i < arr.length; i++) {
+            if (arr[i] > arr[i + 1]) {
+                return false;
+            }
+        }
+        return true;
```

Comment:
You can use `reduce` function for simplification here

Reply:
Thanks. Reworked the code.